### ISSUE TITLE
Customizing colors in columns UI

### ISSUE THREAD

**Issue by colinsullivan** [bram85/topydo#140]
Hello!

Thank you for all of your work!

Is it possible to configure the colors used in the columns UI?

Specifically the highlight color and text changing to black during highlight?

<img width="297" alt="screen shot 2016-09-29 at 9 21 02 am" src="https://cloud.githubusercontent.com/assets/94535/18962530/19692730-8626-11e6-861d-0ca9633afe54.png">

<img width="285" alt="screen shot 2016-09-29 at 9 22 47 am" src="https://cloud.githubusercontent.com/assets/94535/18962584/50a8caf2-8626-11e6-963c-0de34a513fed.png">

**Comment by bram85** [comment]
No, these colors are hardcoded at the moment. But after having a brief look
at the code in shouldn't be too hard to make it configurable.

**Comment by colinsullivan** [comment]
How does this look to you?

I just did the "focus background" colors.

https://github.com/colinsullivan/topydo/commit/a38c763d52c63913f168bcadbb9ea4d41ab5c6c3

**Comment by bram85** [comment]
Can you demo it with a screenshot, please?
Code-wise the patch looks OK, if you make a PR that makes it easier to review, test and merge it.

**Comment by colinsullivan** [comment]
My `.topydo`:

```
...
[columns]
column_width = 79

[colorscheme]
context_color = 6
context_focus_background_color = 8
default_focus_background_color = 8
project_focus_background_color = 8
link_focus_background_color = 8
metadata_focus_background_color = 8
marked_background_color = 16

```

Here is a screenshot:

![screen shot 2016-11-21 at 10 31 10 am](https://cloud.githubusercontent.com/assets/94535/20495848/a7cffb68-afd7-11e6-969b-2371b8659512.png)

Current issue:

Behavior is not correct when I drop one of these new values from the config.  I tried putting the defaults in here: https://github.com/colinsullivan/topydo/commit/ef851a7b83a739e9f8f534ea2e56011eb0bcac20

But this doesn't quite do the trick.

I'm happy to create a pull request with what is there if it would help.

**Comment by colinsullivan** [comment]
On a related note and more of a design choice: I'm not sure anyone would ever want to control these "focus background color" values independently:

```
context_focus_background_color
default_focus_background_color
project_focus_background_color
link_focus_background_color
metadata_focus_background_color
```

Controlling them independently creates square blocks around the different components of the todo.  Perhaps just one `focus_background_color` is more appropriate?

**Comment by bram85** [comment]
Yes, exactly the same thought crossed my mind while I was watching your screenshot. I think a single background color for focus (and another one for a marked item) will do.

**Comment by bram85** [comment]
`light gray` is only valid for urwid, the UI library, topydo's Color class understands `gray`, I believe you're looking for that one.

**Comment by bram85** [comment]
Closing as this was implemented with #151.
